### ISSUE TITLE
How to present monitored parameters?

### ISSUE THREAD

**Issue by maelle** [masalmon/openaq_figures#2]
@RocketD0g other part of the discussion. I won't do anything else than this copy-paste today. ;-)

Regarding the parameters, it's harder to show all info. I've made a barplot of number of stations per parameter, and a histogram of the number of parameters per station. However one could also want to show the variety of monitored parameters depending on countries, or what combination of monitored parameters are the most common. I've made a last figure where one sees the number of locations for each possible combination parameters. It's ugly, but it's worth knowing which ones are the most frequent: it might be worth writing in the text.

**Comment by RocketD0g** [comment]
All of these parameter plots are interesting. And agree, it will, at minimum be interesting to note the most common parameter combinations. 

I wonder: perhaps this plot: https://github.com/masalmon/openaq_figures/blob/master/code_files/figure-html/unnamed-chunk-4-1.png

could be taken and each bar could be separated by some sort of region? Maybe by country, but probably too busy a figure. Maybe by continent, but that could be too coarse - though I lean toward that more than to separate out into 18 countries.

If it is easy, could you share the x axis in elongated form (could be truncated in the y-direction) for this graph or as the raw data in a spreadsheet? https://github.com/masalmon/openaq_figures/blob/master/code_files/figure-html/unnamed-chunk-6-1.png

...the combo frequency is def interesting and would like to see the breakdown.

cc: jflasher, as you're interested.

**Comment by maelle** [comment]
It's so nice to have the conversation on Github! I like living in this time of history.
- Continents are tricky: in which continent is Hawai? And if you get France one day, it is in several continents... I'd have to use lon/lat because of some weird countries! But I see what you mean. I have yet to find good groups.
- Is this spreadsheet what you wanted? https://github.com/masalmon/openaq_figures/blob/master/combination.csv

**Comment by maelle** [comment]
I can share data for any of the plots in csv or xlsx format without much effort, so don't hesitate to ask.

**Comment by maelle** [comment]
@RocketD0g I must say I have no idea for potential groups on the parameters figure. Maybe the only simple message is the info about the most frequent combinations, and maybe you'd want it by country?

**Comment by RocketD0g** [comment]
Yeah, agreed - I think for now the simple message about the most frequent combos, just given as stats, is prob the way to go - probably not even by country at this stage. We can incorporate the general stat in the paper and we can see how it fits.

**Comment by maelle** [comment]
The new file with combinations is here: https://github.com/masalmon/openaq_figures/blob/master/combination.csv

and the figure with the number of locations per parameter is here: https://github.com/masalmon/openaq_figures/blob/master/figures/parameters1.png
